### ISSUE TITLE
Bug: Map series index missing with new record

### ISSUE THREAD

**Issue by ghost** [klokantech/kartenportal-tickets#287]
FROM:zb
The new record http://suche.kartenportal.ch/#id=326348123 (Trinidad 1:25,000 soils) is a map series. There are 38 sheets belonging to that series, for example http://suche.kartenportal.ch/#series&id=326805427 (sheet 72).

The map series index is only visible for just one single sheet when one clicks the appropriate button. The neighbouring sheets are not displayed. However, the Swissbib hierarchy tree looks fine (see https://www.swissbib.ch/Record/326348123/HierarchyTree?hierarchy=326348123&recordID=326348123). 

If you think that the sheet index will be visible after next Sunday's regular import of the complete Swissbib map dump, then you may wait and review the issue next week again. Otherwise, please fix.

**Comment by klokan** [comment]
To be tested now - after the regular import is done. @keosak PLS
Tracking to kartenportal2014

**Comment by keosak** [comment]
@clientportal

This problem was caused by the automatic update failing. We have been using a JSON document at https://test.swissbib.ch/mapportal_all.json to generate statistics about the number of records in each institution. However, the URL now returns status code 404 (Not Found). This crashed the update process and new records didn't enter the database.

I have removed the statistics update from the process for now. What is the new URL for the `mapportal_all.json` document that we should use?
